www/src/components/seo/headSeo.astro

```diff
@@ -1,51 +1,35 @@
 ---
-import { OPEN_GRAPH, SITE, type Frontmatter } from "../../config";
+import { OPEN_GRAPH, SITE } from "../../config";
 import { SITE_URL } from "@utils/siteUrl";
-
-export interface Props {
-  frontmatter?: Frontmatter;
-  type?: string;
-}
-
-const { frontmatter, type } = Astro.props as Props;
-
+ 
 const canonicalUrl = new URL(Astro.url.pathname, Astro.site);
 
-const formattedContentTitle = frontmatter
-  ? `${frontmatter.title} 🚀 ${SITE.title}`
-  : SITE.title;
-
-const ogTitle = frontmatter ? frontmatter.title : SITE.title;
-const ogDescription = frontmatter ? frontmatter.description : SITE.description;
-
-const ogImageData = {
-  title: ogTitle,
-  description: ogDescription,
-  pagePath: Astro.url.pathname,
-};
-
-const imageSrc =
-  frontmatter?.image?.src ??
-  `${SITE_URL}/og?${Object.entries(ogImageData)
-    .map(([key, value]) => `${key}=${value}`)
-    .join("&")}`;
-
-const imageUrl = new URL(imageSrc, Astro.url.origin);
-const imageAlt = frontmatter?.image?.alt ?? OPEN_GRAPH.image.alt;
-
-const ogType = type ?? "article";
+// const imageUrl = SITE_URL+"/api/og"
+const imageUrl = SITE_URL+"/og-image.png"
+console.log(imageUrl)
 ---
 
 <!-- Page Metadata -->
 <link rel="canonical" href={canonicalUrl} />
 <meta name="description" property="og:description" content={SITE.description} />
 
 <!-- OpenGraph Tags -->
-<meta property="og:title" content={formattedContentTitle} />
-<meta property="og:type" content={ogType} />
+<meta property="og:title" content={SITE.title} />
+<meta property="og:description" content={SITE.description} />
 <meta property="og:url" content={canonicalUrl} />
-<meta property="og:locale" content={SITE.defaultLanguage} />
+<!-- <meta property="og:image" content={imageUrl} /> -->
 <meta property="og:image" content={imageUrl} />
-<meta property="og:image:alt" content={imageAlt} />
-<meta property="og:site_name" content={SITE.title} />
- 
+<meta property="og:image:alt" content={OPEN_GRAPH.image.alt} />
+<meta property="og:image:type" content="image/png" />
+<meta property="og:image:width" content="1200" />
+<meta property="og:image:height" content="630" /> 
+
+<!-- Twitter Tags -->
+<meta property="twitter:image" content={imageUrl}></meta>
```

Comment:
I think the Twitter ones should be `<meta name=` rather than using `property=` as I mentioned in my last review -- is this verified to work as a `property=`

re: https://stackoverflow.com/a/66816822

Reply:
<img width="758" alt="Screenshot 2023-12-04 at 9 03 19 PM" src="https://github.com/danstepanov/create-expo-stack/assets/39573679/f9f528c5-2567-434f-9714-686fd50ffe34">

just for the card, site and image right?
found this resources https://www.jafaraziz.com/blog/generate-open-graph-images-with-astro-and-satori/

Reply:
I just changed all the twitter tags from property to name like from https://stackoverflow.com/a/66816822. I'll this the resource above in case it doesn't work as a reference.

Reply:
There is this website to to test what the social image looks like, https://socialsharepreview.com/

It looks like this for the vercel preview, so i guess it works 
<img width="818" alt="Screenshot 2023-12-04 at 9 15 10 PM" src="https://github.com/danstepanov/create-expo-stack/assets/39573679/d3d2c344-3efc-4203-bb62-976459b4847e">